src/mudlet.pro

```diff
@@ -195,14 +194,13 @@ isEmpty( FONT_TEST ) | !equals($$upper(FONT_TEST), "NO" ) {
 }
 
 ######################### Auto Updater setting detection #########,#############
-# Enable the auto updater system by default on supported platforms. unless the
-# environmental variable WITH_UPDATER is already defined and is not
-# set to (case insenstive) "no". This is for Linux packagers and others who will
-# want to handle updates themselves.
-# Note: as WITH_UPDATER could be a number, a string, something else (or not even
-# exist) we need to be careful in checking it exists before doing much else
-# with it. Also as an environmental variable it is tricky to handle unless we
-# read it into a qmake variable first:
+# To remove the built-in updater, set the environment WITH_UPDATER variable to "NO"
```

Comment:
To remove the built-in updater ***on supported platforms***,  set the environment WITH_UPDATER variable to "NO"!

Reply:
There is obviously no need to remove an updater on non-supported platforms...